skype/skype-ps/skype/Enable-CsOnlineSipDomain.md

```diff
@@ -34,6 +34,8 @@ Before running this cmdlet for any SIP domain in a Skype for Business Server dep
 **IMPORTANT**
 If you have more than one on-premises deployment of Skype for Business Server, you *must* ensure SharedSipAddressSpace is disabled in all other Skype for Business Server deployments except the deployment containing the SIP domain that is being enabled. 
 
+Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.  
```

Comment:
```suggestion
Note: If the Tenant is disabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.  
```

Reply:
Sorry, this comment is for the other PR.

Reply:
If the Tenant is enabled for RHM in SFBO - 
either running enable or disable sip domain command - must be run on all the forests to reflect the changes correctly.
Check the last warning in the screenshot:
![image](https://user-images.githubusercontent.com/64554275/114888589-00d7fc00-9e27-11eb-91b6-8a80fa8d55a7.png)